security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisTool.java

```diff
@@ -192,8 +191,8 @@ public void run(CommandLine line, ToolRunningContext context) throws Exception {
         }
 
         SecurityAnalysisParameters parameters = SecurityAnalysisParameters.load();
-        if (line.hasOption(PARAMETERS_FILE)) {
-            Path parametersFile = context.getFileSystem().getPath(line.getOptionValue(PARAMETERS_FILE));
+        if (line.hasOption(PARAMETERS_FILE_OPTION)) {
```

Comment:
Note: you may use the ToolOptions utility class, which wraps command line options in a typed Optional (Optional<Path> here for example).

Reply:
I think this should be done in another PR in that we change all for `ActionSimulatorTool` too.

Reply:
Ok!